src/app/page/MissionDetails/MissionDetailsView.jsx

```diff
@@ -1,88 +1,393 @@
+            {Icon !== undefined ? <Icon color="primary" /> : null}
+            {avatarImage !== undefined ? (
+              <Avatar className={classes.avatar} alt="Volunteer" src={avatarImage} />
+            ) : null}
+          </Grid>
+          <Grid item xs={11}>
+            {content}
+          </Grid>
+        </React.Fragment>
+      );
+    })}
+  </Grid>
+);
+
+const MissionDetailsContent = ({ description, classes }) => (
+  <Box>
+    <H5 align="left" color="textSecondary">
+      Mission Type
+    </H5>
+    <Body2 align="left" className={classes.missionTypeText} color="textPrimary">
+      {description}
+    </Body2>
+  </Box>
+);
+
+const MissionDetailsButton = ({
+  type,
+  volunteerForMission,
+  startMission,
+  markMissionAsDelivered,
+}) => {
+  switch (type) {
+    case MissionStatus["unassigned"]:
+      return (
+        <StyledButton
+          color="primary"
+          variant="contained"
+          disableElevation
+          onClick={volunteerForMission}
+        >
+          Accept Mission
+        </StyledButton>
+      );
+    case MissionStatus["tentative"]:
+    case MissionStatus["assigned"]:
+      return (
+        <StyledButton color="primary" variant="contained" disableElevation onClick={startMission}>
+          Start Mission
+        </StyledButton>
+      );
+    case MissionStatus["started"]:
+      return (
+        <StyledButton
+          color="primary"
+          variant="contained"
+          disableElevation
+          onClick={markMissionAsDelivered}
+        >
+          Mark Mission as Delivered
+        </StyledButton>
+      );
+    case MissionStatus["delivered"]:
+    case MissionStatus["done"]:
+      return null;
+    default:
+      return (
+        <StyledButton color="primary" variant="contained" disableElevation>
+          Loading..
+        </StyledButton>
+      );
+  }
+};
+
+const MissionDetailsPickUpDeliveryHeader = ({ header, classes }) => (
+  <Body2 align="left" className={classes.deliveryDetailsHeader} color="textPrimary">
+    {header}
+  </Body2>
+);
 
 const MissionDetailsPage = ({
   mission,
+  volunteer,
   volunteerForMission,
-  markMissionAsCompleted,
+  startMission,
+  markMissionAsDelivered,
+  unassignFromMission,
   userUnverifiedPopupOpen,
   setUserUnverifiedPopupOpen,
-  cords,
   history,
 }) => {
+  //mock data
+  mission = {
+    ...mission,
+    fundedStatus: MissionFundedStatus["fundedbydonation"],
+    status: MissionStatus["unassigned"],
+    pickUpWindow: "1:30 PM",
+    pickUplocation: "123 Strawberry Ln, VA 22201",
+    deliveryWindow: "2:30–3:30 PM",
+    deliverylocation: "123 Strawberry Ln, VA 22201",
+    recipientName: "John Doe",
+    recipientPhoneNumber: "(123) 456-7890",
+  };
+  volunteer = {
+    ...volunteer,
+    profileName: "Jane",
+    avatar: "https://qodebrisbane.com/wp-content/uploads/2019/07/This-is-not-a-person-2-1.jpeg",
+  };
+
+  const classes = useStyles();
+  const title = mission.title || null;
```

Comment:
I am confused. Should this be deleted? Why is it so similar to MissionDetailsCard.jsx?